horace_core/sqw/@Experiment/get_efix.m

```diff
@@ -0,0 +1,5 @@
+function efix = get_efix(obj)
+% Return array of incident energies from all runs, contributing to
+% experiment
+efix = arrayfun(@(x)x.efix(:)',obj.expdata_,'UniformOutput',false);
+efix = [efix{:}];
```

Comment:
```suggestion
efix = arrayfun(@(x)x.efix(:)',obj.expdata_);
```
Or if that doesn't work
```suggestion
efix = arrayfun(@(x)x.efix(:)',obj.expdata_,'UniformOutput',false);
efix = cell2mat(efix);
```

Reply:
I've added comment, why the design is as provided.